client/src/modules/list/components/Items/model/actions.js

```diff
@@ -357,7 +363,7 @@ export const restoreItem = (listId, itemId, name) => dispatch =>
     });
 
 export const deleteItem = (listId, itemId, name) => dispatch =>
-  patchData(`/api/lists/${listId}/delete-item/${itemId}`)
+  deleteData(`/api/lists/${listId}/delete-item/${itemId}`)
```

Comment:
In order to be more REST-ful, you should not include action names in the path. For example, proper path in this case would be `DELETE /api/lists/${listId}/items/${itemId}` or `DELETE /api/lists/${listId}/${itemId}`.